packages/jest-runtime/src/shouldInstrument.js

```diff
@@ -84,6 +84,20 @@ export default function shouldInstrument(
     return false;
   }
 
+  if (
+    config.setupFiles &&
+    config.setupFiles.some(setupFile => setupFile === filename)
+  ) {
+    return false;
+  }
+
+  if (
+    config.setupFilesAfterEnv &&
```

Comment:
`setupFilesAfterEnv` and `setupFiles` are normalized to return empty array, so this extra check is not necessary

Reply:
@thymikee I think they are not normalized for unit tests, just checked and got `TypeError: Cannot read property 'some' of undefined`, do you want me to leave the check here or provide a default value somewhere?

Reply:
Oh, that's because `shouldInstrument` test file is not typed. Wanna try to add `@flow` pragma to it and make it use normalized config as it actually expects? If that's too tedious, just pass default values for these specific tests.

Reply:
yeah sure, I'll try to take care of this

Reply:
@thymikee I've added the pragma and types, how would I make it use the normalized config tho?

Reply:
`jest-runtime` has a dependency on `jest-config` which provides `normalize`. See: https://github.com/facebook/jest/blob/74ae342b28de957c0d3cf42545cfcf17bd172e4f/packages/jest-resolve-dependencies/src/__tests__/dependency_resolver.test.js#L11-L33

Reply:
ok, now you should be good to remove these extra checks (here and wherever suitable)

Reply:
The thing is that currently, test cases overwrite the default config when they pass their own config object to `testShouldInstrument` so it still throws a `Cannot read property 'some' of undefined` error. How about making the `Cannot read property 'testShouldInstrument` in `should_instrument.test.js` merge the default config and the one from args?

Reply:
`shouldInstrument` expects every `config` to be normalized. As I said, it's probably to much of a hassle and not in scope of this PR, so I'm good with leaving this as is (although it seems like an area for some small speedup, so definitely worth following up later)

Reply:
@SimenB comment added